bitbots_vision/bitbots_vision/params.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python3
+
+from rcl_interfaces.msg import ParameterDescriptor, FloatingPointRange, IntegerRange, ParameterType
+
+class ParameterGenerator:  # TODO own file
+  def __init__(self):
+    self.param_cache = []
+
+  def declare_params(self, node):
+    for param in self.param_cache:
+      node.declare_parameter(*param)
```

Comment:
```suggestion
    node.declare_parameter('', self.param_cache)
```

Reply:
See here: https://github.com/ros2/rclpy/blob/master/rclpy/rclpy/node.py#L361 and the function above.